app/models/book_suggestion.rb

```diff
@@ -0,0 +1,4 @@
+class BookSuggestion < ApplicationRecord
+  belongs_to :user
```

Comment:
Are we requiring this relation? check http://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html